src/libs/metadata.c

```diff
@@ -81,10 +81,8 @@ static gboolean _is_leave_unchanged(GtkTextView *textview)
 static gchar *_get_buffer_text(GtkTextView *textview)
 {
   GtkTextBuffer *buffer = gtk_text_view_get_buffer(textview);
-  GtkTextIter start;
```

Comment:
Hm, first time I see this file in this commit? Accidentd from some merge, or intentional? I don't see anything related to adobe_coeff here...

Reply:
A mess on my side after checking another PR :)